instana/instrumentation/django/middleware.py

```diff
@@ -29,19 +29,27 @@ def __init__(self, get_response=None):
         super(InstanaMiddleware, self).__init__(get_response)
         self.get_response = get_response
 
+    def _extract_custom_headers(self, span, headers, format):
+        try:
+            if agent.options.extra_http_headers is not None:
```

Comment:
This check should be inverted for an early return and should go touside of the `try` block.